src/server/local/index.js

```diff
@@ -64,29 +62,6 @@ export default server => {
 
       sendDataFile(res, dataFilePath, next);
     })
-    .get(frontPageDataPath, async ({ params }, res, next) => {
-      const { service, variant } = params;
-
-      const dataFilePath = constructDataFilePath({
-        pageType: 'frontpage',
-        service,
-        variant,
-      });
-
-      sendDataFile(res, dataFilePath, next);
-    })
-    .get(homePageDataPath, async ({ params }, res, next) => {
```

Comment:
We do not use the home page data route any more right? In constructDataFilePath it uses the path /index.json, which we don't use.

Reply:
https://www.bbc.com/mundo/index.json this is not our 404 page

Reply:
Confusingly, we serve the file named index.json within data/mundo/homePage as the local homepage data route http://localhost:7080/mundo.json 🤯